pages/glossary/visual-reference-words.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Visual Reference Words
+key: visual-reference-words
+---
+
```

Comment:
Please put "WIP" in the title, or make this PR a draft until this is ready for a review.